examples/upload_dataset_metadata.py

```diff
@@ -0,0 +1,166 @@
+# example of using API to upload annotations and QA status from local copy of dataset
```

Comment:
This doesn't seem like a super-useful script. What would be more useful is a script that takes a dataset id, and an arbitrary json file, and e.g. syncs the Conservator annotation QA status with the QA status in the file. Also, the name of this script is misleading, as it does not upload dataset metadata, it only makes some changes to the annotations.